### PR TITLE
10.0 fix ir attachment url getting mimetype

### PR DESCRIPTION
**Pull request by rbikbov**

### CODE REVIEW

File: ir_attachment_url/models/binary_fields.py [hunk 1]
```diff
@@ -1,9 +1,32 @@
 # -*- coding: utf-8 -*-
-from odoo import fields
+# Copyright 2018 Rafis Bikbov <https://www.it-projects.info/team/RafiZz>
```
Comment (yelizariev) [inline ir_attachment_url/models/binary_fields.py:3]: Add the rest developers, please

**Review by yelizariev — CHANGES_REQUESTED** [review]
Also, add updating version and add changelog into the first commit

File: ir_attachment_url/models/binary_fields.py (hunk 1 above)
Comment (rbikbov) [reply ir_attachment_url/models/binary_fields.py:3]: Done.

File: ir_attachment_url/doc/changelog.rst [hunk 2]
```diff
@@ -1,3 +1,8 @@
+`1.1.7`
+-------
+
+- **Fix**  When a link to a picture that does not have an extension is written in a binary field, its mimetype is not determined, which leads to an "binascii.Error: decoding with base64 codec failed (Error: Incorrect padding)"
```
Comment (yelizariev) [inline ir_attachment_url/doc/changelog.rst:4]: :+1:

File: ir_attachment_url/models/binary_fields.py [hunk 3]
```diff
@@ -21,13 +45,17 @@ def write(self, records, value):
         if value and image.is_url(value):
             with records.env.norecompute():
                 if value:
+                    mimetype = get_mimetype_by_url(value)
+                    index_content = mimetype.split('/')[0]
```
Comment (yelizariev) [inline ir_attachment_url/models/binary_fields.py]: :-1:

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: ir_attachment_url/models/binary_fields.py (hunk 3 above)
Comment (rbikbov) [reply ir_attachment_url/models/binary_fields.py]: Now it is better?

File: ir_attachment_url/models/binary_fields.py [hunk 4]
```diff
@@ -21,13 +46,18 @@ def write(self, records, value):
         if value and image.is_url(value):
             with records.env.norecompute():
                 if value:
+                    mimetype, content = get_mimetype_by_url(value)
```
Comment (yelizariev) [inline ir_attachment_url/models/binary_fields.py]: The method is called "get mimetype", but it also returns content

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: ir_attachment_url/models/binary_fields.py (hunk 4 above)
Comment (rbikbov) [reply ir_attachment_url/models/binary_fields.py]: Renamed.

**Review by yelizariev — APPROVED** [review]

**Pull request merged**